src/transformers/generation_tf_utils.py

```diff
@@ -2119,6 +2127,7 @@ def sample(
         max_length: Optional[int] = None,
         pad_token_id: Optional[int] = None,
         eos_token_id: Optional[int] = None,
+        seed: Optional[int] = None,
```

Comment:
This is a list/tuple of two ints, right?
```suggestion
        seed: Optional[Sequence[int, int]] = None,
```

Reply:
yes, correct (adding Patrick's suggestions now)

Reply:
You might have to `from typing import Sequence` at the top too, sorry about that!